torch/csrc/functorch/init.cpp

```diff
@@ -490,6 +488,7 @@ void initFuncTorchBindings(PyObject* module) {
   m.def("push_dynamic_layer_stack", [](DynamicLayer layer) -> int64_t {
     return pushDynamicLayer(std::move(layer));
   });
+  // NOLINTNEXTLINE(bugprone-unused-raii)
```

Comment:
Why is only this one problematic?

Reply:
Known false positive bug with clang-tidy and pybind11 if you instantionate a py::class without defining any methods on it. If it wasn't for the global side effects of adding the casters for it, it would be a NOOP which would be a bug. However, since creating the object globally affects the caster translation dict, it is necessary to do even if the RAII for creating / cleaning up the class do nothing.